packages/react-examples/src/react-button/SplitButton/SplitButton.stories.tsx

```diff
@@ -16,30 +17,34 @@ const menuProps = {
   ],
 };
 
+const Menu = (defaultSlotType: string, props: MinimalMenuProps) => {
```

Comment:
Is this a render function `(Component, props) => JSX`? If so, is `string` the right type?

Reply:
This is the expected function call but I think I can see a way of only passing `props` here. I'll investigate.

Reply:
Made it so that only `props` need to be passed.